maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,422 @@
+                            agent_spans = _agent_spans.get()
+                            if agent_name not in agent_spans:
+                                span_id = str(uuid4())
+                                agent_spans = agent_spans.copy()
+                                agent_spans[agent_name] = trace.span({
+                                    "id": span_id,
+                                    "name": f"Agent: {agent_name}",
+                                    "tags": {
+                                        "agent_type": (
+                                            trace_tags["agent_type"]
+                                            if "agent_type" in trace_tags
+                                            else "unknown"
```

Ruff (0.12.2)

Use `trace_tags.get("agent_type", "unknown")` instead of an `if` block

Replace with `trace_tags.get("agent_type", "unknown")`

(SIM401)